c++/src/Reader.hh

```diff
@@ -146,6 +146,7 @@ namespace orc {
     uint64_t firstStripe;
     uint64_t currentStripe;
     uint64_t lastStripe; // the stripe AFTER the last one
+    uint64_t currentInitedStripe;
```

Comment:
The naming is a little bit casual, maybe processingStripe?

Reply:
Done.